tests/client/test_secure_data_hosting.py

```diff
@@ -0,0 +1,210 @@
+import hashlib
+import os
+import requests
+
+from clarifai_grpc.grpc.api import service_pb2_grpc, service_pb2, resources_pb2
+from tests.common import (
+    TRUCK_IMAGE_URL,
+    TRAVEL_IMAGE_URL,
+    BEER_VIDEO_URL,
+    both_channels,
+    raise_on_failure,
+    wait_for_inputs_upload,
+)
+
+req_session = requests.Session()
+
+PAT = os.environ.get("CLARIFAI_PAT_KEY_SECURE_HOSTING")
+API_KEY = os.environ.get("CLARIFAI_API_KEY_SECURE_HOSTING")
+SESSION_TOKEN = os.environ.get("CLARIFAI_SESSION_TOKEN_SECURE_HOSTING")
```

Comment:
do we use session token?

Reply:
for the app owner, yes (we also use this in the e2es)

Reply:
Oh sorry, I disabled them when I was trying to debug the thumbnail issue. These should be re-enabled.